### PR TITLE
Add hyp3lib to conda-forge

### PR DESCRIPTION
**Pull request by jhkennedy**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] ~~If static libraries are linked in, the license of the static library is packaged.~~
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jhkennedy** [comment]
I'm listed as a maintainer and am willing to be listed there

File: recipes/hyp3lib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    - setuptools_scm >=3.4
+    - wheel
+  run:
+    - boto3
+    - gdal
+    - imageio
+    - importlib_metadata
+    - lxml
+    - matplotlib-base
+    - netCDF4
```
Comment (scopatz) [inline recipes/hyp3lib/meta.yaml:68]: Dependencies should be lowercase

Comment (jhkennedy) [reply recipes/hyp3lib/meta.yaml]: Right! Thanks.

**Comment by jhkennedy** [comment]
@conda-forge/help-python I'm not sure what's happening with `win_64` -- I'm getting a lot of `WARNING: liefldd: failed to ensure_binary` and `pyldd.py failed to process`. 

When I [run tests locally](https://conda-forge.org/docs/maintainer/adding_pkgs.html#staging-test-locally) with
```sh
CONFIG=win64 ./.circleci/run_docker_build.sh
```
everything works fine

**Comment by scopatz** [comment]
noarch recipes only need to work on Linux

**Comment by jhkennedy** [comment]
> noarch recipes only need to work on Linux

Great, them I'm happy! I don't actually care about `win_64` either way

**Comment by scopatz** [comment]
Thanks!

**Comment by jhkennedy** [comment]
> Thanks!

No thank you!

**Pull request merged**